### PR TITLE
Feature/#20 distinguish concordia buildings

### PR DESCRIPTION
**Pull request by razine-bensari**
Added all markers and polygons delimitation for campus buildings. It is a tedious and long task. Everything is hard coded since no api is available.

### CODE REVIEW

**Comment by Alessandro100** [comment]
Notes:
- styling on iOS is not correct [BUG]
- when tapping on an icon, an action menu at the bottom of the screen should appear, not a label above the icon (not sure if this is in the scope of the user story)
- pins in each building should be pins and not circles

Overall, I like the way you organized the constants, I appreciate the effort that went into drawing the polygons. Fix the comments, the icon tap and the pin design and Ill merge it in

File: App.tsx [hunk 1]
```diff
@@ -1,19 +1,29 @@
 import React, { Component } from 'react';
-import { StyleSheet, View, Dimensions } from 'react-native';
-import MapView, { PROVIDER_GOOGLE } from 'react-native-maps';
+import { StyleSheet, View, Dimensions, Alert, Text } from 'react-native';
+import MapView, {Marker, Polygon, PROVIDER_GOOGLE} from 'react-native-maps';
 import CampusToggleButton from './components/CampusToggleButton';
+import { buildingPolygons } from "./constants/CampusPolygons";
+import { markers } from './constants/CampusMarkers';
```
[the remark this answers is not in the document]
Comment (razine-bensari) [reply App.tsx]: changed!

File: App.tsx [hunk 2]
```diff
@@ -1,19 +1,29 @@
...
 
 class App extends Component {
+  constructor() {
+    // @ts-ignore
```
[the remark this answers is not in the document]
Comment (razine-bensari) [reply App.tsx]: removed

File: App.tsx [hunk 3]
```diff
@@ -24,21 +34,54 @@ class App extends Component {
...
+      return (
+        <View style={styles.container}>
+          <View style={styles.search}/>
+          <CampusToggleButton setMapLocation={this.setMapLocation}/>
+          <MapView provider={PROVIDER_GOOGLE} style={styles.mapStyle} region={region}>
+              {this.state.polygons.map((polygon, index) => (
+                  <View key={index}>
+                      <Polygon
+                          coordinates={polygon}
+                          strokeColor='rgb(163, 9, 9)'
```
[the remark this answers is not in the document]
Comment (razine-bensari) [reply App.tsx]: refactored

**Pull request merged**